source/tyr/migrations/env.py

```diff
@@ -62,7 +62,7 @@ def exclude_tables_from_config(config_):
     return tables
 
 
-exclude_tables = exclude_tables_from_config(config.get_section('alembic:exclude'))
+exclude_tables = exclude_tables_from_config(config.get_pt_section('alembic:exclude'))
```

Comment:
how come this line is changed? :D

Reply:
Good catch ! That was comming from PyCharm's refactor tool :/